packages/dev/core/src/AudioV2/webAudio/webAudioEngine.ts

```diff
@@ -160,7 +160,8 @@ export class _WebAudioEngine extends AudioEngineV2 {
 
     /** @internal */
     public get state(): AudioEngineV2State {
-        return this._audioContext.state;
+        // Always return "running" for OfflineAudioContext so sound `play` calls work while the context is suspended.
+        return <any>this._audioContext instanceof OfflineAudioContext ? "running" : this._audioContext.state;
```

Comment:
If this is checked frequently, we should keep track of instanceof result for perf ?

Reply:
Good call! Thanks.